Formula/easy-rsa.rb

```diff
@@ -0,0 +1,34 @@
+class EasyRsa < Formula
+  desc "CLI utility to build and manage a PKI CA."
+  homepage "https://github.com/OpenVPN/easy-rsa"
+  url "https://github.com/OpenVPN/easy-rsa/archive/3.0.1.tar.gz"
+  sha256 "a1fff75a27ea7da3f37fbfed715633f55b9ca25f5b14cac38e525c5c995e68ae"
+
+  option "with-openssl", "Use the openSSL provided by homebrew"
+  depends_on "openssl" => :optional
+
+  def install
+    # We don't use the installer from the package, as it doesn't organize things
+    # well for brew install.
+    inreplace "easyrsa3/easyrsa" do |s|
+      s.gsub! /set_var EASYRSA		"\$PWD"/, \
+              "set_var EASYRSA		\"#{libexec}\""
+      s.gsub! /set_var EASYRSA_PKI	.*/, \
+              "set_var EASYRSA_PKI	\"#{etc}/pki\""
+      if build.with? "openssl"
+        openssl_path = Formula["openssl"].opt_bin/"openssl"
+        ohai "Using OpenSSL from #{openssl_path}"
+        s.gsub! /set_var EASYRSA_OPENSSL	openssl/, \
+                "set_var EASYRSA_OPENSSL		\"#{openssl_path}\""
+      end
+    end
+    bin.install "easyrsa3/easyrsa"
+    libexec.install "easyrsa3/openssl-1.0.cnf", "easyrsa3/x509-types"
+  end
+
+  test do
+    # Create a stub PKI directory in the test sandbox.
+    Kernel.system({ "EASYRSA_PKI" => "./pki" }, "#{bin}/easyrsa --help")
```

Comment:
No, not yet. We still support the older Rubies, at least until https://github.com/Homebrew/brew/pull/513 lands & we get broader user feedback on that working as-expected.